packages/flutter/test/material/carousel_test.dart

```diff
@@ -26,12 +26,12 @@ void main() {
       ),
```

Comment:
Do we have tests for the new weighted constructor here that are horizontal, horizontal+reverse, vertical, and verical+reverse?

Reply:
Added some more tests for vertical, reverse, and itemSnapping combinations.